src/zeit/content/volume/browser/form.py

```diff
@@ -87,23 +83,38 @@ def setUpWidgets(self, *args, **kw):
             self.widgets['volume'].setRenderedValue(settings.default_volume)
 
     def add(self, object):
```

Comment:
Können wir nicht generell `object` in `volume` umbenennen? Find ich lesbarer und ich mag es nicht, eingebaute Namen als Variablen zu verwenden.